libcron/include/libcron/Cron.h

```diff
@@ -184,8 +284,8 @@ namespace libcron
         return d;
     }
 
-    template<typename ClockType, typename LockType>
-    size_t Cron<ClockType, LockType>::tick(std::chrono::system_clock::time_point now)
+    template<typename ClockType, typename LockType, typename NameScheduleMapType>
+    size_t Cron<ClockType, LockType, NameScheduleMapType>::tick(std::chrono::system_clock::time_point now)
     {
         tasks.lock_queue();
         size_t res = 0;
```

Comment:
Lets rename this to `execution_count` and use it in the for-loop at line 346 instead of i. We can then use it to check if we need to sort and can also use it as the return value. One less variable in this method!